recipe_sha: 849663d3d24092a6e1a9138c19394056f5b5ed60c2673d60ed8d7cc4fd7cc607  # The hash of the recipe's meta.yaml at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |2-
        |                      ^~~~~
  sam_record.cpp: In function 'std::string format_sam_flags(uint16_t)':
  sam_record.cpp:42:56: error: 'read_paired' is not a member of 'samflags'
     42 |   oss << "read_paired: " << check(the_flags, samflags::read_paired) << '\n'
        |                                                        ^~~~~~~~~~~
  sam_record.cpp:44:37: error: 'read_pair_mapped' is not a member of 'samflags'
     44 |       << check(the_flags, samflags::read_pair_mapped) << '\n'
        |                                     ^~~~~~~~~~~~~~~~
  sam_record.cpp:45:58: error: 'read_unmapped' is not a member of 'samflags'
     45 |       << "read_unmapped: " << check(the_flags, samflags::read_unmapped) << '\n'
        |                                                          ^~~~~~~~~~~~~
  sam_record.cpp:46:58: error: 'mate_unmapped' is not a member of 'samflags'
     46 |       << "mate_unmapped: " << check(the_flags, samflags::mate_unmapped) << '\n'
        |                                                          ^~~~~~~~~~~~~
  sam_record.cpp:47:52: error: 'read_rc' is not a member of 'samflags'
     47 |       << "read_rc: " << check(the_flags, samflags::read_rc) << '\n'
        |                                                    ^~~~~~~
  sam_record.cpp:48:52: error: 'mate_rc' is not a member of 'samflags'
     48 |       << "mate_rc: " << check(the_flags, samflags::mate_rc) << '\n'
        |                                                    ^~~~~~~
  sam_record.cpp:50:37: error: 'template_first' is not a member of 'samflags'
     50 |       << check(the_flags, samflags::template_first) << '\n'
        |                                     ^~~~~~~~~~~~~~
  sam_record.cpp:51:58: error: 'template_last' is not a member of 'samflags'
     51 |       << "template_last: " << check(the_flags, samflags::template_last) << '\n'
        |                                                          ^~~~~~~~~~~~~
  sam_record.cpp:52:58: error: 'secondary_aln' is not a member of 'samflags'
     52 |       << "secondary_aln: " << check(the_flags, samflags::secondary_aln) << '\n'
        |                                                          ^~~~~~~~~~~~~
  sam_record.cpp:53:58: error: 'below_quality' is not a member of 'samflags'
     53 |       << "below_quality: " << check(the_flags, samflags::below_quality) << '\n'
        |                                                          ^~~~~~~~~~~~~
  sam_record.cpp:54:58: error: 'pcr_duplicate' is not a member of 'samflags'
     54 |       << "pcr_duplicate: " << check(the_flags, samflags::pcr_duplicate) << '\n'
        |                                                          ^~~~~~~~~~~~~
  sam_record.cpp:56:37: error: 'supplementary_aln' is not a member of 'samflags'
     56 |       << check(the_flags, samflags::supplementary_aln);
        |                                     ^~~~~~~~~~~~~~~~~
  sam_record.cpp: In function 'std::ostream& operator<<(std::ostream&, const sam_rec&)':
  sam_record.cpp:79:19: error: 'const class sam_rec' has no member named 'flags'
     79 |              << r.flags << '\t'
        |                   ^~~~~
  sam_record.cpp:81:19: error: 'const class sam_rec' has no member named 'pos'
     81 |              << r.pos << '\t'
        |                   ^~~
  sam_record.cpp:82:41: error: 'const class sam_rec' has no member named 'mapq'
     82 |              << static_cast<unsigned>(r.mapq) << '\t'
        |                                         ^~~~
  sam_record.cpp:85:19: error: 'const class sam_rec' has no member named 'pnext'; did you mean 'rnext'?
     85 |              << r.pnext << '\t'
        |                   ^~~~~
        |                   rnext
  sam_record.cpp: In constructor 'sam_rec::sam_rec(const std::string&)':
  sam_record.cpp:102:18: error: 'flags' was not declared in this scope
    102 |         qname >> flags >> rname >> pos >> will_become_mapq >>
        |                  ^~~~~
  sam_record.cpp:102:36: error: 'pos' was not declared in this scope; did you mean 'pow'?
    102 |         qname >> flags >> rname >> pos >> will_become_mapq >>
        |                                    ^~~
        |                                    pow
  sam_record.cpp:103:27: error: 'pnext' was not declared in this scope; did you mean 'rnext'?
    103 |         cigar >> rnext >> pnext >> tlen >> seq >> qual))
        |                           ^~~~~
        |                           rnext
  sam_record.cpp:105:7: error: 'mapq' was not declared in this scope
    105 |   if (mapq > 255)
        |       ^~~~
  sam_record.cpp:107:3: error: 'mapq' was not declared in this scope
    107 |   mapq = static_cast<uint8_t>(will_become_mapq);
        |   ^~~~
  make[3]: *** [Makefile:487: sam_record.o] Error 1
  make[3]: Leaving directory '$SRC_DIR/src/smithlab_cpp'
  make[2]: *** [Makefile:370: all] Error 2
  make[2]: Leaving directory '$SRC_DIR/src/smithlab_cpp'
  make[1]: *** [Makefile:540: all-recursive] Error 1
  make[1]: Leaving directory '$SRC_DIR'
  make: *** [Makefile:380: all] Error 2
  Traceback (most recent call last):
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2558, in build
      utils.check_call_env(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/utils.py", line 404, in check_call_env
      return _func_defaulting_env_to_os_environ("call", *popenargs, **kwargs)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/utils.py", line 380, in _func_defaulting_env_to_os_environ
      raise subprocess.CalledProcessError(proc.returncode, _args)
  subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'errexit', '/opt/conda/conda-bld/preseq_1734244836153/work/conda_build.sh']' returned non-zero exit status 2.

  The above exception was the direct cause of the following exception:

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-build", line 11, in <module>
      sys.exit(execute())
    File "/opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py", line 589, in execute
      api.build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 209, in build
      return build_tree(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3712, in build_tree
      packages_from_this = build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2566, in build
      raise BuildScriptException(str(exc), caused_by=exc) from exc
  conda_build.exceptions.BuildScriptException: Command '['/bin/bash', '-o', 'errexit', '/opt/conda/conda-bld/preseq_1734244836153/work/conda_build.sh']' returned non-zero exit status 2.
# Last 100 lines of the build log.
